aten/src/ATen/Declarations.cwrap

```diff
@@ -915,6 +915,8 @@
     - Long
     - Int
     - Short
+    - Byte
```

Comment:
nit: can we just remove the whole `types:` part, as I think it defaults to dispatching to all types when it's not present?